tests/test_config_io.py

```
@@ -0,0 +1,323 @@
+    from graph.config_io import list_gateway_models
+
+    models, err = list_gateway_models("", "key")
+    assert models == []
+    assert "api_base" in err
+
+
+def test_list_gateway_models_http_error(monkeypatch):
+    from graph import config_io
+
+    fake_client = MagicMock()
+    fake_client.__enter__ = lambda self: fake_client
```

Ruff (0.15.10)

[warning] Unused lambda argument: `self`

(ARG005)